terraform/context_plan2_test.go

```diff
@@ -54,11 +54,6 @@ resource "test_object" "a" {
 			t.Fatalf("expected Create action for missing %s, got %s", c.Addr, c.Action)
 		}
 	}
-
-	_, diags = ctx.Apply()
-	if diags.HasErrors() {
-		t.Fatal(diags.Err())
-	}
```

Comment:
This slipped into the change from the file I had open already, but I left it since it's a bit of code that should have been deleted. `Apply` shouldn't be called in the plan test, and does not check anything.